test/Unit/Durable/DurableTimerTests.cs

```diff
@@ -0,0 +1,187 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Test.Durable
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using Microsoft.Azure.Functions.PowerShellWorker.Durable;
+    using Moq;
+    using Xunit;
+
+    public class DurableTimerTests
+    {
+        private const int LongIntervalSeconds = 5;
+        private const int ShortIntervalSeconds = 3;
+        private static readonly DateTime startTime = DateTime.UtcNow;
```

Comment:
Naming convention for non-const data members: prefix with underscore (e.g. `_startTime`)